src/main/java/me/ryanhamshire/GriefPrevention/FlatFileDataStore.java

```diff
@@ -320,6 +320,11 @@ else if (this.getSchemaVersion() == 0)
                         List<String> containerNames = Arrays.asList(line.split(";"));
                         containerNames = this.convertNameListToUUIDList(containerNames);
 
+                        //fifth line is list of players who can access creatures
+                        line = inStream.readLine();
+                        List<String> creatureNames = Arrays.asList(line.split(";"));
+                        creatureNames = this.convertNameListToUUIDList(creatureNames);
```

Comment:
Name list should already be a UUID list - this conversion is for legacy data. Doesn't need to be done.